contents/tutorials/zapier-surveys.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to send survey responses to Zapier
+date: 2025-05-01
+author:
+  - lior-neu-ner
+showTitle: true
+sidebar: Docs
+tags:
+  - surveys
+---
+
+It's useful to send survey responses to [Zapier](https://zapier.com/). This way you can automatically trigger workflows in thousands of other apps, such as creating tickets in your help desk, or updating your CRM with customer feedback.
```

Comment:
```suggestion
It can be useful to send survey responses to [Zapier](https://zapier.com/). This way you can automatically trigger workflows in thousands of other apps, such as creating tickets in your help desk or updating your CRM with customer feedback.
```